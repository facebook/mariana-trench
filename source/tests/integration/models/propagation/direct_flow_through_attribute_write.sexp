(method (private) "LFlow;.flow:(LSink;LSource;)V"
 (
  LINE(1)
  (invoke-virtual (v3) "LSource;.source:()LData;")
  (move-result-object v0)

  LINE(2)
  (iput-object v0 v5 "LData;.field:LData;")

  LINE(3)
  (invoke-virtual (v2 v5) "LSink;.sink:(LData;)V")
  (return-void)
 )
)
