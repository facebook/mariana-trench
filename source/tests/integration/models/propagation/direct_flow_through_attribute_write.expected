{
   "metadata" : {
      "@generated" : true
   },
   "models" : [
      {
         "method" : "LFlow;.flow:(LSink;LSource;)V",
         "position" : {
            "line" : 1,
            "path" : "Flow.java"
         }
      }
   ]
}
